assets/xml/objects/object_sk2.xml

```diff
@@ -0,0 +1,55 @@
+<Root>
+    <File Name="object_sk2">
+        <DList Name="skeltonMeshHead" Address="0x53D0"/>
+        <DList Name="skeltonMeshThirdVertebrae" Address="0x5720"/>
+        <DList Name="skeltonMeshRightShoulderBone" Address="0x57B8"/>
```

Comment:
are these names prefixed with skeleton as in sk2? Id do stalfos just cause that can get confused very quickly with the skeleton from skelanime.
Im sure these names are far from final and not the point of the PR, I just dont want to cause confusion as we start to think of standards for naming this stuff.

Also `skelton_mesh_fourth_vertebrae` below doesnt follow the same camel casing as the rest of the entries

Reply:
A bunch of these names were either borrowed from the 3DS version or the Cloudmodding wiki.

Reply:
And yeah, the names are by no means final.